### PR TITLE
Make command palette string transatables

### LINKED ISSUE #59132: Not all command palette strings are translatable.
### Description

Hello,

I noticed that not all command palette strings are translatable.

I noted the following:
- [Edit template](https://github.com/WordPress/gutenberg/blob/1c7f3a8f97dcc47b6c1b61f633ccb0cf362b8268/packages/edit-site/src/hooks/commands/use-edit-mode-commands.js#L78)
- [Delete template](https://github.com/WordPress/gutenberg/blob/1c7f3a8f97dcc47b6c1b61f633ccb0cf362b8268/packages/edit-site/src/hooks/commands/use-edit-mode-commands.js#L148)
- [Delete template part](https://github.com/WordPress/gutenberg/blob/1c7f3a8f97dcc47b6c1b61f633ccb0cf362b8268/packages/edit-site/src/hooks/commands/use-edit-mode-commands.js#L153)
- [Reset template](https://github.com/WordPress/gutenberg/blob/1c7f3a8f97dcc47b6c1b61f633ccb0cf362b8268/packages/edit-site/src/hooks/commands/use-edit-mode-commands.js#L124)
- [Reset template part](https://github.com/WordPress/gutenberg/blob/1c7f3a8f97dcc47b6c1b61f633ccb0cf362b8268/packages/edit-site/src/hooks/commands/use-edit-mode-commands.js#L129)

Please see my Pull request to fix this issue : https://github.com/WordPress/gutenberg/pull/59133

### Step-by-step reproduction instructions

1. Visit a locale WordPress translate project and search for one of the command listed below ([example in French](https://translate.wordpress.org/projects/wp/dev/fr/default/?filters%5Bterm%5D=Delete+template+part&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated_or_rejected_or_changesrequested_or_old&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc))
3. No string is available to translation

### Screenshots, screen recording, code snippet

_No response_

### Environment info

- WordPress 6.4 (French (fr_FR))

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

### PR DESCRIPTION
**Pull request by MarieComet**
See the related issue : https://github.com/WordPress/gutenberg/issues/59132

### CODE REVIEW

**Comment by huzaifaalmesbah** [comment]
I See Your PR Looks Good. Thanks for PR

File: packages/edit-site/src/hooks/commands/use-edit-mode-commands.js [hunk 1]
```diff
@@ -119,14 +119,16 @@ function useManipulateDocumentCommands() {
 	if ( isTemplateRevertable( template ) && ! isEditingPage ) {
 		const label =
 			template.type === TEMPLATE_POST_TYPE
-				? /* translators: %1$s: template title */
+				?
 				  sprintf(
-						'Reset template: %s',
+					        /* translators: %1$s: template title */
+						__( 'Reset template: %s' ),
```
Comment (t-hamano) [inline packages/edit-site/src/hooks/commands/use-edit-mode-commands.js]:
```suggestion
						/* translators: %s: template title */
						__( 'Reset template: %s' ),
```

File: packages/edit-site/src/hooks/commands/use-edit-mode-commands.js [hunk 2]
```diff
@@ -119,14 +119,16 @@ function useManipulateDocumentCommands() {
...
 						decodeEntities( template.title )
 				  )
-				: /* translators: %1$s: template part title */
+				:
 				  sprintf(
-						'Reset template part: %s',
+					  	/* translators: %1$s: template part title */
+						__( 'Reset template part: %s' ),
```
Comment (t-hamano) [inline packages/edit-site/src/hooks/commands/use-edit-mode-commands.js]:
```suggestion
						/* translators: %s: template part title */
						__( 'Reset template part: %s' ),
```

File: packages/edit-site/src/hooks/commands/use-edit-mode-commands.js [hunk 3]
```diff
@@ -143,14 +145,16 @@ function useManipulateDocumentCommands() {
 	if ( isTemplateRemovable( template ) && ! isEditingPage ) {
 		const label =
 			template.type === TEMPLATE_POST_TYPE
-				? /* translators: %1$s: template title */
+				?
 				  sprintf(
-						'Delete template: %s',
+					  	/* translators: %1$s: template title */
+						__( 'Delete template: %s' ),
```
Comment (t-hamano) [inline packages/edit-site/src/hooks/commands/use-edit-mode-commands.js]:
```suggestion
						/* translators: %s: template title */
						__( 'Delete template: %s' ),
```

File: packages/edit-site/src/hooks/commands/use-edit-mode-commands.js [hunk 4]
```diff
@@ -143,14 +145,16 @@ function useManipulateDocumentCommands() {
...
 						decodeEntities( template.title )
 				  )
-				: /* translators: %1$s: template part title */
+				:
 				  sprintf(
-						'Delete template part: %s',
+					  	/* translators: %1$s: template part title */
+						__( 'Delete template part: %s' ),
```
Comment (t-hamano) [inline packages/edit-site/src/hooks/commands/use-edit-mode-commands.js]:
```suggestion
						/* translators: %s: template part title */
						__( 'Delete template part: %s' ),
```

**Review by t-hamano — COMMENTED** [review]
Great catch!

I believe these changes are correct, but it seems to be failing the lint check because it's indented with spaces instead of tabs. If you commit the suggestions below, it should be resolved.

At the same time, I also fixed the placeholder format (`%1$s`), which was originally incorrect.

**Comment by MarieComet** [comment]
Hi @t-hamano 
Thanks for the review.
I commited your suggestions and fixed [another incorrect placeholder format in comment](https://github.com/WordPress/gutenberg/pull/59133/commits/43f9b48d5ebb6342858ad8cdd667e0594c4ee8c8).

**Review by t-hamano — APPROVED** [review]
LGTM! GitHub Actions is all ✅. Thanks for your great work!

**Comment by getdave** [comment]
I just cherry-picked this PR to the cherry-pick-beta-2 branch to get it included in the next release: e14c1d95d9

**Pull request merged**